src/components/faqTemplates/ExpandableWithCategories.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SectionContainer } from '@lib/Containers'
+import { Spacers } from '@lib/Helpers'
+
+const FaqAnswerContainer = styled.div`
+  display: ${props => (props.shouldDisplay ? 'block' : 'none')};
+  overflow: hidden;
+  margin: auto;
+  margin-bottom: 2%;
+`
+
+const Accordion = styled.button`
+  width: 100%;
+  cursor: pointer;
+  text-align: left;
+  outline: none;
+  border: 3px solid black;
+  border-radius: 10px;
+  background-color: transparent;
+`
+
+const FaqQuestionContainer = styled.div``
+
+const ExpandableFaq = ({ question, answer }) => {
+  const [isExpanded, toggleExpansion] = useState(false)
+
+  return (
+    <>
+      <Accordion onClick={() => toggleExpansion(!isExpanded)}>
+        <FaqQuestionContainer>{question}</FaqQuestionContainer>
+        <FaqAnswerContainer shouldDisplay={isExpanded}>{answer}</FaqAnswerContainer>
+      </Accordion>
+    </>
+  )
+}
+
+const TitleImg = styled.img`
+  margin-bottom: ${props => props.marginBottom};
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+`
+
+// FAQ Section with two columns with layout:
+// General    Logistics
+//            Teams & Projects
+const FaqSection = props => {
+  const [generalBin, setGeneralBin] = useState([])
+  const [logisticsBin, setLogisticsBin] = useState([])
+  const [teamsBin, setTeamsBin] = useState([])
+
+  const distributeFaq = faq => {
+    faq.forEach(item => {
+      switch (item.category) {
+        case 'General':
+          setGeneralBin([...generalBin, item])
+          break
+        case 'Logistics':
+          setLogisticsBin([...logisticsBin, item])
+          break
+        case 'Teams & Projects':
+          setTeamsBin([...teamsBin, item])
+          break
+        default:
+          throw new Error(`FAQ category ${item.category} not found`)
+      }
+    })
+  }
+
+  const renderFaq = list =>
```

Comment:
Could we make this more component like so usage looks more like `<Faq list={xxx} />` instead of `renderFaq(xxx)`

Reply:
That sounds good, should we refactor common components in each section to src/lib/<section> so other templates could use it as well? e.g. under src/lib/Faq/Faq.jsx?

Reply:
I'm choosing not to do that because I'd also have to export ExpandableFaq as well and not every template uses that.

Reply:
It's also very little code